oaknational/Oak-Web-Application
# LESQ-62 Feat/examboard unit listing

## Description

add exam board to unit listing title if exist

## Issue(s)

Fixes #

## How to test

can only be testing when corresponding db-tools ticket is merged

1. Go to https://deploy-preview-1737--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/history-secondary-ks4-aqa/units
3. You should see examboard title in programme title

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
